### ISSUE TITLE
Rozszerzenie statystyk o najczęściej używane technologie w danym stacku

### ISSUE THREAD

**Issue by pawelszalw** [Czy-Jest-Eldorado/feature-requests#31]
Cześć! Przede wszystkim dzięki za wspaniałą wyszukiwarkę ofert pracy!

Pytanie:
Czy rozważaliście rozszerzenie obecnych statystyk o informacje na temat najczęściej używanych technologii w danej kategorii ? Widziałbym to tak, że wybierając np. Testing w statystykach można byłoby odczytać, że 30% ogłoszeń wspomina np. o selenium, 25% o python, 40% o javie itd. 
Mogłoby to pomóc juniorom lepiej planować naukę, a seniorom dać pewność, że ich aktualny projekt umożliwia im rozwój i nie hamuje ich postępu.

**Comment by jkobus** [comment]
Tak, myślę, że coś takiego jest do zrobienia. Ogólnie planowałem wykresy obecnych trendów na rynku – co zyskuje na popularności, a co traci, gdzie liczba ofert rośnie, a gdzie spada. Twój pomysł jest bardzo zbliżony, więc na pewno będę go mieć na uwadze. Dzięki!

**Comment by jkobus** [comment]
Niedawno wprowadziliśmy funkcjonalność, która pokazuje właśnie takie dane - najczęściej występujące technologie w obrębie danej kategorii oraz ich powiązania z ofertami.

Możesz ją przetestować tutaj:
https://czyjesteldorado.pl/praca/kategoria/testing
